python/graphframes/graphframe.py

```diff
@@ -207,15 +207,26 @@ def bfs(self, fromExpr, toExpr, edgeFilter=None, maxPathLength=10):
 
     # Standard algorithms
 
-    def connectedComponents(self):
+    def connectedComponents(self, algorithm = "graphframes", checkpointInterval = 1,
+                            broadcastThreshold = 1000000):
```

Comment:
if the threshold is too large, it may become a python `long` and fail to get passed to the jvm. This is probably not going to happen anytime soon.